custom/src/main/java/org/kiwix/kiwixmobile/custom/main/CustomReaderFragment.kt

```diff
@@ -95,22 +95,43 @@ class CustomReaderFragment : CoreReaderFragment() {
       loadUrlWithCurrentWebview(pageUrl)
     } else {
       openObbOrZim()
-      restoreLastOpenedTab()
+      manageExternalLaunchAndRestoringViewState()
```

Comment:
so this function is in custom and app? probably should be in core then?

Reply:
Yes, I think these can be more similar and moved to core.